internal/config/global.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+type Global struct {
+	namespace         string
+	operateInFIPSMode bool
+	version           string
+}
+
+type Option func(*Global)
+
+// WithNamespace sets both TargetNamespace, DefaultTelemetryNamespace and ManagerNamespace to the given value.
+// TODO: Split into separate options.
+func WithNamespace(namespace string) Option {
+	return func(g *Global) {
+		g.namespace = namespace
+	}
+}
+
+func WithOperateInFIPSMode(enable bool) Option {
```

Comment:
lets remove the enable parameter. it is by default false and the option just enables it